tests/library/modernizr.spec.ts

```diff
@@ -30,17 +30,33 @@ async function checkFeatures(name: string, context: any, server: any) {
   }
 }
 
-it('safari-14-1', async ({ browser, browserName, platform, server, headless, isMac }) => {
+it('Safari Desktop', async ({ browser, browserName, platform, server, headless, isMac }) => {
   it.skip(browserName !== 'webkit');
   it.skip(browserName === 'webkit' && platform === 'darwin' && parseInt(os.release(), 10) === 22, 'Modernizr uses WebGL which is not available in macOS-13 - https://bugs.webkit.org/show_bug.cgi?id=278277');
```

Comment:
Perhaps we can update `expected` instead, now with the new version of Modernizr?

Reply:
Modernizr crashes with `WebGL: context lost` and is not able to generate a report when WebGL is not available / on our macOS-13 bots. So keeping it skipped for now.